extensions/molecule/default/molecule.yml

```
@@ -2,6 +2,10 @@
 driver:
   name: docker
 platforms:
+  - name: openwrt_x86_64-25.12.0-rc1
+    image: openwrt/rootfs:x86_64-25.12.0-rc1
+    command: sh -c "sleep infinity"
```

Since it is only one simple command, you can run it directly. ;-)
```suggestion
    command: sleep infinity
```